scraper.rb

```diff
@@ -3,43 +3,29 @@
 # frozen_string_literal: true
 
 require 'pry'
+require 'require_all'
 require 'scraped'
 require 'scraperwiki'
 
+require_rel 'lib'
+
 # require 'open-uri/cached'
 # OpenURI::Cache.cache_path = '.cache'
 require 'scraped_page_archive/open-uri'
 
-def noko_for(url)
-  Nokogiri::HTML(open(url).read)
+def scrape(h)
+  url, klass = h.to_a.first
+  klass.new(response: Scraped::Request.new(url: url).response)
 end
 
-def scrape_list(url)
-  noko = noko_for(url)
-  noko.xpath('.//ul[@id="leftnavigation"]/li[contains(.,"States Members")]/following-sibling::li[@class="child"]//a/@href').each do |li|
-    scrape_mp(li.text)
+def scrape_list(term, url)
+  (scrape url => MembersPage).member_urls.each do |mem_url|
+    data = (scrape mem_url => MemberPage).to_h.merge(district: 'Alderney',
+                                                     party:    'Independent',
+                                                     term:     term)
+    ScraperWiki.save_sqlite(%i(id term), data)
```

Comment:
As discussed in Leeds, unless there's a good reason, let's aim for consistency by building up the `data` and then saving it in one shot.